docs/skopeo-delete.1.md

```diff
@@ -19,24 +19,46 @@ $ docker exec -it registry /usr/bin/registry garbage-collect /etc/docker-distrib
 
 ```
 
+## OPTIONS
+
 **--authfile** _path_
 
-  Path of the authentication file. Default is ${XDG_RUNTIME\_DIR}/containers/auth.json, which is set using `skopeo login`.
-  If the authorization state is not found there, $HOME/.docker/config.json is checked, which is set using `docker login`.
+Path of the authentication file. Default is ${XDG_RUNTIME\_DIR}/containers/auth.json, which is set using `skopeo login`.
+If the authorization state is not found there, $HOME/.docker/config.json is checked, which is set using `docker login`.
+
+**--creds** _username[:password]_
+
+Credentials for accessing the registry.
+
+**--cert-dir** _path_
+
+Use certificates at _path_ (*.crt, *.cert, *.key) to connect to the registry.
+
+**--daemon-host** _host_
 
-**--creds** _username[:password]_ for accessing the registry.
+Use docker daemon host at _host_ (`docker-daemon:` transport only)
 
-**--cert-dir** _path_ Use certificates at _path_ (*.crt, *.cert, *.key) to connect to the registry.
+**--help**, **-h**
 
-**--tls-verify** _bool-value_ Require HTTPS and verify certificates when talking to container registries (defaults to true).
```

Comment:
Just to make sure this isn’t lost, there’s been a fairly long discussion / research around the `--tls-verify` flag, https://github.com/containers/skopeo/pull/1350#discussion_r661757345 .

Reply:
Oof, yeah, "long" is right.  Would you mind opening an issue about it? I feel that might be a better way to ensure the discussion isn't lost, since addressing it in this (or that #1350) PR seems onerous.

Reply:
The thing is, this PR is mostly blocked on that discussion — without fixing `--help`, enforcing correspondence between `--help` and man pages would make the state of documentation worse.

But contrary to the bleak mood of most of that discussion, there might be a fairly clean way forward now: https://github.com/containers/skopeo/pull/1369 . It definitely needs an independent review, but looks promising.